mqtt/build/linux/buildImage.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# This script copies the iodedged executable files that go into the azureiotedge-iotedged image,
+# for each supported arch (x86_64, arm32v7, arm64v8).
+# It then publishes executable files along with their corresponding dockerfiles to the publish directory,
+# so that buildImage.sh can build the container image.
+
+set -e
+###############################################################################
+# Define Environment Variables
+###############################################################################
+
+ARCH=$1
+DOCKER_REGISTRY=$2
+DOCKER_USERNAME=$3
+DOCKER_PASSWORD=$4
+DOCKER_IMAGEVERSION=${BUILD_BUILDNUMBER:=$5}
+
+if [[ "$ARCH" == "amd64" ]]; then
+    ARCH="amd64"
+    DOCKER_IMAGENAME="azureiotedge/mqttd-linux-amd64"
+    TARGET="x86_64-unknown-linux-musl"
+    DOCKERFILE="../../docker/linux/amd64/Dockerfile"
```

Comment:
I believe you need to change this path. Scripts in the pipeline run from root directory that's why our scripts have several  ENV https://github.com/Azure/iotedge/blob/master/scripts/linux/buildEdgelet.sh#L22-L24